static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -245,3 +245,341 @@
+        margin-bottom: 0;
+        width: auto;
+        float: none;
+    }
+
+    .block-share--gallery {
+        position: relative;
+
+        .block-share__item--facebook,
+        .block-share__item--twitter,
+        .block-share__item--pinterest {
+            background-color: rgba(0, 0, 0, 0);
+            border: 1px solid $neutral-2;
+
+            &:hover {
+                background-color: rgba(0, 0, 0, 0);
+            }
+
+            &.inline-icon, .inline-icon {
+                fill: $neutral-3;
+            }
+        }
+    }
+
+    .social-icon, .block-share__item {
+        background-color: rgba(0, 0, 0, 0);
+        border: 1px solid $neutral-2;
+
+        &:hover {
+            background-color: rgba(0, 0, 0, 0);
+        }
+
+        &.inline-icon, .inline-icon {
+            fill: $neutral-3
+        }
+    }
+
+    .gallery {
+        list-style: none;
+        margin: 0;
+    }
+
+    .gallery__item {
+        figure {
+            display: flex;
+            flex-direction: column-reverse;
+        }
+
+        // On mobile the images should be 100% of the width of the screen
+        @include mq($until: phablet) {
+            margin-left: -$gs-gutter;
+            margin-right: -$gs-gutter;
+        }
+
+        @include mq($until: mobileLandscape) {
+            margin-left: -$gs-gutter / 2;
+            margin-right: -$gs-gutter / 2;
+        }
+
+        @include mq(tablet) {
+            position: relative;
+            border-top: 0;
+            padding: 0;
+
+            figure {
+                flex-direction: row;
+            }
+        }
+    }
+
+    .gallery__figcaption {
+        @include fs-textSans(1);
+        padding-top: $gs-baseline / 4;
+        overflow: visible;
+        max-height: 100%;
+        position: relative;
+        border: 0;
+        width: inherit;
+
+        @include mq($until: desktop) {
+            margin-bottom: $gs-baseline * 2;
+        }
+
+        @include mq($until: phablet) {
+            padding: $gs-gutter;
+        }
+
+        @include mq($until: mobileLandscape) {
+            padding: $gs-gutter / 2;
+        }
+
+        @include mq(tablet) {
+            @include fs-textSans(2, true);
+            position: absolute;
+            width: gs-span(3);
+            top: 0;
+            right: auto;
+            margin-left: -(gs-span(3));
+        }
+    }
+
+    .gallery__caption {
+        color: $neutral-4;
+        margin-bottom: 0;
+
+        @include mq(tablet) {
+            padding-right: $gs-gutter / 2;
+        }
+
+        a {
+            color: $neutral-4;
+            border-bottom: 1px solid #767676;
+
+            &:hover {
+                border-bottom: 1px solid $neutral-7;
+                text-decoration: none;
+            }
+        }
+
+        > strong:first-child {
+            @include fs-headline(2);
+            color: $neutral-5;
+            display: block;
+            margin-bottom: $gs-baseline/2;
+
+            & + br {
```

Comment:
I have been playing around with it, but I don't think there is. The problem is more of a tooling problem than anything else. We don't get properly structured data back so the caption looks like this:

![image](https://cloud.githubusercontent.com/assets/8774970/14820325/1501d12a-0bbe-11e6-8a4b-12bca4ddbc0e.png)

The first `br` is display none and the "title" (what is in the `strong` tags) is display block with the correct margin-bottom. 

If we make all `br`s display none, then in long captions the editors wont have the ability to make "paragraphs" or breaks between sentences.

If we leave it and take out the display block in the title, we don't get the correct spacing between the title and the rest of the caption.

The best thing to do is change the tooling to allow us to template this properly. I am happy to talk to the tools team about this, but it won't be for this PR.